bullet-featherstone/src/KinematicsFeatures.cc

```diff
@@ -125,6 +132,8 @@ FrameData3d KinematicsFeatures::FrameDataRelativeToWorld(
     }
     else if (isCollision)
       data.pose = data.pose * collisionPoseOffset;
+    else if (isJoint)
```

Comment:
Is this necessary? I think it's handled above and codecov shows it's not covered by tests.